package.json

```diff
@@ -18,6 +18,7 @@
     "express": "^4.17.1",
     "jquery": "^3.5.1",
     "jsonwebtoken": "^8.5.1",
+    "moment": "^2.29.1",
```

Comment:
Adding this so we can calculate the current mileage based on how much time has passed since they registered a car